mysql-async/src/main/java/com/github/jasync/sql/db/mysql/util/CapabilityFlag.kt

```diff
@@ -0,0 +1,14 @@
+@file:Suppress("unused")
+
+package com.github.jasync.sql.db.mysql.util
+
+enum class CapabilityFlag(val value: Int) {
```

Comment:
I think this is a more typesafe change and makes it easier to reason about these ints in the system.

Reply:
great!